data/remnant.txt

```diff
@@ -131,12 +192,19 @@ phrase "remnant"
 		"Lost"
 		"Changed"
 		"Alive"
+		"Restored"
+		"Preserved"
+		"Renewed"
+		"Reborn"
+		"Ascending"
+		"Soaring"
+		"Enduring"
 
 
 
 shipyard "Remnant"
 	"Starling"
-	"Albatross"
```

Comment:
What I think you should do is add the new license with its outfit, but make its image "unknown" for the time being. That way the only thing missing would be the outfit image, instead of having Remnant ships be missing from their shipyards.

Reply:
Amusingly, that's what I started out doing, except I used the current Remnant license as a placeholder just so there'd be *something* there, with the intention that it be replaced, and that was one of the few issues that MZ actually commented on.

I've also been under the impression that everything needs proper images to be merged - no more merging stuff with blanks. Along the same lines, no-one seems to be leaping at making a new license. So while I agree with this change, it is basically reverting to the way I originally had it.

Reply:
Alright, I've added the Remnant Capital License, using the "Unknown" image.